app/services/order_fees_handler.rb

```diff
@@ -20,6 +20,8 @@ def recreate_all_fees!
       create_line_item_fees!
       create_order_fees!
     end
+
+    order.update!
```

Comment:
Calling `order.update!` here (in `#recreate_all_fees!`) is useful, it means we can stop manually calling `order.update!` in various places where we call `order.recreate_all_fees!`.